ec2-spot-elastic-inference/EI_launch_template.json.bak

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do we really need to commit bak files to the repo?   I think you can remove this.